src/index.js

```diff
@@ -29,7 +29,7 @@ class Timeout {
 
 const _setTimeout = (...args) => { return new Timeout(...args); };
 
-const _clearTimeout = (timer) => { return timer.close(); };
+const _clearTimeout = (timer) => { timer && timer.close && timer.close(); };
```

Comment:
I'm always leery of using `instanceOf`... do you have a strong case for it other than removing one `&&`?